features/FEATURE_PSA/TARGET_TFM/src/os_wrapper_cmsis_rtos_v2.c

```diff
@@ -75,21 +103,25 @@ uint32_t os_wrapper_semaphore_delete(void *handle)
         return OS_WRAPPER_ERROR;
     }
 
+    free(handle);
+
     return OS_WRAPPER_SUCCESS;
 }
 
 void *os_wrapper_mutex_create(void)
 {
+    osRtxMutex_t *mutex;
+
+    mutex = (osRtxMutex_t *)malloc(osRtxMutexCbSize);
+    if(mutex == NULL){
+        return NULL;
+    }
+
     const osMutexAttr_t attr = {
         .name = NULL,
-        .attr_bits = osMutexPrioInherit, /* Priority inheritance is recommended
-                                          * to enable if it is supported.
-                                          * For recursive mutex and the ability
-                                          * of auto release when owner being
-                                          * terminated is not required.
-                                          */
-        .cb_mem = NULL,
-        .cb_size = 0U
+        .attr_bits = osMutexRecursive | osMutexPrioInherit | osMutexRobust,
+        .cb_mem = mutex,
+        .cb_size = osRtxMutexCbSize
```

Comment:
Consider using the Mbed OS C++ Mutex class instead.

Reply:
Fixed in a later commit.